### ISSUE TITLE
Manage grayscale images

### ISSUE THREAD

**Issue by miguelgfierro** [miguelgfierro/sciblog_support#49]
we can create fake RGB images from 1 channel images by replicating the channels:

```python
data_transform = transforms.Compose([
    transforms.ToTensor(),
    transforms.Lambda(lambda x: torch.cat([x, x, x], 0))
])
```
[source](https://discuss.pytorch.org/t/transfer-learning-grayscale-image-size-and-activation-function/9953/2)

**Comment by miguelgfierro** [comment]
When using MNIST, they load the image as grayscale: https://github.com/pytorch/vision/blob/master/torchvision/datasets/mnist.py#L73
 img = Image.fromarray(img.numpy(), mode='L')

**Comment by miguelgfierro** [comment]
With gray images using transform `transforms.Normalize([0.485, 0.456, 0.406], [0.229, 0.224, 0.225])`:
```
Epoch 1/25
----------
train Loss: 0.0110 Acc: 0.1633
val Loss: 0.0098 Acc: 0.3128

Epoch 2/25
----------
train Loss: 0.0089 Acc: 0.3836
val Loss: 0.0070 Acc: 0.5857

Epoch 3/25
----------
train Loss: 0.0070 Acc: 0.5464
val Loss: 0.0048 Acc: 0.7230

Epoch 4/25
----------
train Loss: 0.0058 Acc: 0.6209
val Loss: 0.0036 Acc: 0.7910

Epoch 5/25
----------
train Loss: 0.0051 Acc: 0.6662
val Loss: 0.0030 Acc: 0.8266

Epoch 6/25
----------
train Loss: 0.0045 Acc: 0.6947
val Loss: 0.0025 Acc: 0.8509

Epoch 7/25
----------
train Loss: 0.0041 Acc: 0.7183
val Loss: 0.0022 Acc: 0.8665

Epoch 8/25
----------
train Loss: 0.0041 Acc: 0.7245
val Loss: 0.0022 Acc: 0.8683

Epoch 9/25
----------
train Loss: 0.0039 Acc: 0.7362
val Loss: 0.0022 Acc: 0.8708

Epoch 10/25
----------
train Loss: 0.0039 Acc: 0.7357
val Loss: 0.0022 Acc: 0.8719

Epoch 11/25
----------
train Loss: 0.0039 Acc: 0.7317
val Loss: 0.0021 Acc: 0.8762

Epoch 12/25
----------
train Loss: 0.0039 Acc: 0.7351
val Loss: 0.0021 Acc: 0.8731

Epoch 13/25
----------
train Loss: 0.0038 Acc: 0.7422
val Loss: 0.0020 Acc: 0.8795

Epoch 14/25
----------
train Loss: 0.0038 Acc: 0.7394
val Loss: 0.0020 Acc: 0.8795

Epoch 15/25
----------
train Loss: 0.0039 Acc: 0.7365
val Loss: 0.0020 Acc: 0.8790

Epoch 16/25
----------
train Loss: 0.0038 Acc: 0.7433
val Loss: 0.0020 Acc: 0.8783

Epoch 17/25
----------
train Loss: 0.0038 Acc: 0.7399
val Loss: 0.0020 Acc: 0.8783

Epoch 18/25
----------
train Loss: 0.0038 Acc: 0.7401
val Loss: 0.0020 Acc: 0.8801

Epoch 19/25
----------
train Loss: 0.0039 Acc: 0.7330
val Loss: 0.0020 Acc: 0.8795

Epoch 20/25
----------
train Loss: 0.0039 Acc: 0.7335
val Loss: 0.0020 Acc: 0.8785

Epoch 21/25
----------
train Loss: 0.0038 Acc: 0.7384
val Loss: 0.0020 Acc: 0.8813

Epoch 22/25
----------
train Loss: 0.0038 Acc: 0.7408
val Loss: 0.0020 Acc: 0.8798

Epoch 23/25
----------
train Loss: 0.0038 Acc: 0.7399
val Loss: 0.0020 Acc: 0.8788

Epoch 24/25
----------
train Loss: 0.0038 Acc: 0.7399
val Loss: 0.0020 Acc: 0.8806

Epoch 25/25
----------
train Loss: 0.0037 Acc: 0.7460
val Loss: 0.0020 Acc: 0.8813
Training complete in 15m 59s
Best val Acc: 0.881330
```

**Comment by miguelgfierro** [comment]
With gray images not using transform.
```

```

**Comment by miguelgfierro** [comment]
There is no significant difference between the 2 cases
